code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -16,14 +16,19 @@
 
 	var/faction = list("ashwalker")
 	var/meat_counter = 6
+	var/datum/team/ashwalkers/ashies
 
 /obj/structure/lavaland/ash_walker/Initialize()
 	.=..()
-	START_PROCESSING(SSprocessing, src)	
+	ashies = new /datum/team/ashwalkers(goal = src)
```

Comment:
This is all good, but doesn't the ruin start with few mapped in eggs already present ? These will each create new team instead of sharing one.